Clinical-Genomics/scout
# Gene manual inheritance edit in gene panel

fix #1335 and a bug

**How to test**:
1. Install the branch on stage and go to a gene panel. 
1. Go to the list of included genes and click on 'edit'
1. You should reach the gene-edit panel:
![image](https://user-images.githubusercontent.com/28093618/66646444-70fc3500-ec26-11e9-9e1e-400a983b4e66.png)
1. It should be possible to add custom inheritance models for this gene
1. The available inheritance models should be different from those that are now on prod. These models are specified in #1335
1. Try to add one or more inheritance models from the select as well as one or more custom inheritance models (comma separated) for this gene
1. Update the panel and make sure that all the inheritance models are shown in the gene list of the panel page, like this:
![image](https://user-images.githubusercontent.com/28093618/66646585-dcde9d80-ec26-11e9-8035-667d0729d4f2.png)


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
